packages/stubGenerator/languages/javascript/src/domain/VarType.ts

```diff
@@ -0,0 +1,17 @@
+import * as R from 'ramda';
+
+export type VarType = 'float' | 'int' | 'string' | 'word';
+
+export const isAllOfType = (type: VarType) =>
+  R.pipe(
+    R.map((x: string[]) => R.last(x) as VarType),
+    R.all(R.equals(type)),
+  );
+
+export const replaceLongsWithInt = ([name, type]: string[]) =>
+  type === 'long' ? [name, 'int'] : [name, type];
```

Comment:
I guess we can write it better with `id` function.

Reply:
hmm
I tried but it doesn't look good
```suggestion
export const replaceLongsWithInt = R.cond([
  [R.pipe(typeOf, R.equals('long')), R.pipe(nameOf, n => [n, 'int'])],
  [R.T, R.identity],
]);
// type === 'long' ? [name, 'int'] : [name, type];
```
IMO the original one was better 

what do u think?

Reply:
SO let's keep it as it is.